src/main/scala/scalaz/stream/process1.scala

```diff
@@ -540,6 +540,13 @@ object process1 {
       case \/-(a)   => emit(a)
     }
 
+  def stateScan[S, A, B](init: S)(f: A => State[S, B]): Process1[A, B] = {
+    await1[A] flatMap { a =>
```

Comment:
I guess the type inference *would* work here.  I rather prefer the `await1 flatMap` style, but I don't mind either way.